achievements/security.md

```diff
@@ -1 +1,14 @@
+# Symfony Security
+
+On the subject of security we had some great discussions and research about the state of the
+Symfony Security component as it is now. I has not received much love over the past years and
```

Comment:
I -> It

Reply:
I haz also not received much love over the past year 😢

Reply:
Thanks for the review @OskarStark , was fixed by 95a5db1